app/assets/javascripts/followers.js

```diff
@@ -0,0 +1,13 @@
+$(document).on('ajax:success', '#new_follower', function(e, data, status, xhr) {
```

Comment:
'xhr' is defined but never used.<br>'status' is defined but never used.<br>'data' is defined but never used.<br>'e' is defined but never used.